cfn-update-script/cfn-template-update.py

```diff
@@ -149,7 +163,7 @@ def get_arpio_token(account_id, username, password):
     return token
 
 
-def query_environments(token:str, arpio_account:str)->list[SyncPair]:
```

Comment:
You could go back to the `list[]` syntax with Python 3.12.  :)

Reply:
Might as well do it right across the board tbh